receiver/flinkmetricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,319 @@
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.memory.managed.used:
+    enabled: true
+    description: The amount of managed memory currently used.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.memory.managed.total:
+    enabled: true
+    description: The total amount of managed memory.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.jvm.threads.count:
+    enabled: true
+    description: The total number of live threads.
+    unit: "{threads}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.jvm.garbage_collector.collection.count:
+    enabled: true
+    description: The total number of collections that have occurred.
+    unit: "{collections}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: [ garbage_collector_name ]
+  flink.jvm.garbage_collector.collection.time:
+    enabled: true
+    description: The total time spent performing garbage collection.
+    unit: ms
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: [ garbage_collector_name ]
+  flink.jvm.class_loader.classes_loaded:
+    enabled: true
+    description: The total number of classes loaded since the start of the JVM.
+    unit: "{classes}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.job.restart.count:
+    enabled: true
+    description: The total number of restarts since this job was submitted, including full restarts and fine-grained restarts.
+    unit: "{restarts}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.job.last_checkpoint.time:
+    enabled: true
+    description: The end to end duration of the last checkpoint.
+    unit: ms
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.job.last_checkpoint.size:
+    enabled: true
+    description: The total size of the last checkpoint.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.job.checkpoints.count:
+    enabled: true
+    description: The number of checkpoints by type.
+    unit: "{checkpoints}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: [ checkpoint ]
```

Comment:
Based on the description of the `checkpoint` attribute, `The number of checkpoints in progress, completed or that failed.`, I think this needs to be split into two metrics.
1. Monotonic count of completed/failed
2. Non-monotonic count of in-progress

Reply:
Are you aware that you can define a job then start a job instance multiple times in parallel. I think aggregating all in-progress jobs in that case can be meaningful.

Reply:
I see. Makes sense. updated